### PR TITLE
Add transform to realtimetrains folder

### LINKED ISSUE #25: Transform RealTimeTrain data to retrieve train data
### Task

- [x] Extract arrivals and departures  
- [x] Extract cancellations  
- [x] Extract stations

### User Story

As a developer, I want to transform the RealTimeTrain data so that I can load relevant data into the database and measure performance.

### PR DESCRIPTION
**Pull request by Zhi-704**
Finished transformation step of the ETL pipeline for realtime trains API. Removes unnecessary keys from the API and filters only for train data. Changed a few things in extract to accommodate for missing database, but is an easy change. Close #25.

### CODE REVIEW

File: realtime_trains/transform.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    'isPublicCall',
+    'realtimeArrivalNextDay',
+    'realtimeDepartureNextDay',
+    'platform',
+    'platformConfirmed',
+    'platformChanged'
+]
+SERVICE_CRITERIA = {
+    "serviceType": "train"
+}
```
Comment (FarihaChoudhury) [inline realtime_trains/transform.py:31]: Can you explain your reasoning behind removing these keys, as opposed to creating a new dict with the keys you want?

File: .util/pylint_scores.json [hunk 2]
```diff
@@ -1 +1 @@
-{"avg_score": 10.0}
\ No newline at end of file
+{"avg_score": 9.94}
```
Comment (FarihaChoudhury) [inline .util/pylint_scores.json]: this should be 10!

File: realtime_trains/transform.py (hunk 1 above)
Comment (Zhi-704) [reply realtime_trains/transform.py:31]: Occasionally, when the service is canceled or is any other status besides CALL, additional keys are added and therefore if I used an exclusive approach with the keys in the dictionary, there's a chance that this additional information will not be included.

File: .util/pylint_scores.json (hunk 2 above)
Comment (Zhi-704) [reply .util/pylint_scores.json]: No

Comment (Zhi-704) [reply .util/pylint_scores.json]: Done

File: realtime_trains/transform.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    'tiploc',
+    'country',
+    'system'
+]
+SERVICE_REMOVE_KEYS = [
+    "trainIdentity",
+    "runningIdentity",
+    "isPassenger"
+]
+SERVICE_LOC_DETAILS_REMOVE_KEYS = [
```
Comment (fm1psy) [inline realtime_trains/transform.py:17]: Good use of constants here to make data filtering easier

**Review by fm1psy — DISMISSED** [review]
This looks good, good job

**Review by fm1psy — APPROVED** [review]

**Pull request merged**